style.css

```diff
@@ -156,4 +156,151 @@ ul {
       }
     }
   }
+
+  #works {
```

Comment:
Good work implementing the works section

- [ ] As this is a requirement, this section should be implemented with `grid`. Grid is an interesting and really useful property to make layouts with CSS. Sometimes will be useful `flex` and other times `grid`, it's for this that you need to implement both of them in this project, to show that you know about how to use them 💯